docker-compose.yml

```diff
@@ -1,61 +1,78 @@
 version: '3'
 
 services:
-
-# API
+# PY_API
   server:
-    build: 
-      context: ./packages/api
-      target: final 
-    container_name: api
-    env_file:
-      - ".env"
-    depends_on: 
-      - client
-    expose: 
-      - 8000
-    ports:
-      - 8000:8000
-    command: /api
-    volumes:
-      - ./data/certs:/app/certs
-    networks:
-      - front-back
-
-# WEB
-  client:
```

Comment:
This container is the one hosting the nginx server. Therefore this is the first point of contact when making a request. You can't test the reverse-proxy without having this up and running.